### DISCUSSION TITLE
am i doing this right?

### DISCUSSION THREAD

**Discussion by aviv82** [lab-antwerp-1/home#63] [Q&A] [answered]
made a [PR](https://github.com/lab-antwerp-1/pc-group2/pull/1) after running the code through npm "lint:md" "lint:css" "format" "spell-check" "accessibility" and "validate:html". 

i had to make a few changes and thought it all went well but apparently not.. i am not sure what i should do to resolve this error. please help!

<img width="807" alt="Screen Shot 2021-10-11 at 4 39 29 AM" src="https://user-images.githubusercontent.com/89319835/136726198-2f95fd62-7a4b-4d6a-90f2-da7cbe5e52b2.png">

<img width="658" alt="Screen Shot 2021-10-11 at 4 38 54 AM" src="https://user-images.githubusercontent.com/89319835/136726212-861e6bc0-f399-44b4-abcb-187e825fd41b.png">

**Comment by bermarte** [comment, marked as answer]
Reading the error, it appears that validate should find .html files inside the `/pages` directory but cannot find any.

**Reply by talmurshidi** [reply]
I think they should create empty files in each folder to avoid this kind of errors. I know it is not making sense.

**Reply by aviv82** [reply]
Thanks guys! it worked 🤩
